resources/lang/vi/validation.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => 'The :attribute must be accepted.',
```

Comment:
Convention: trước và sau `=>` có 1 space e nhé
VD: `'accepted' =>  'The :attribute must be accepted.',`--> Fix cả file e nhé